sftp-app-starters-source-common/src/main/java/org/springframework/cloud/stream/app/sftp/source/SftpHeaders.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.sftp.source;
```

Comment:
The rule is: if we have separate jars, they can't share the same package.
So, if you extract into the `sftp-app-starters-source-common`, consider to add a `common` package before `source`, I think...